apps/hash-integration-worker/src/linear-activities.ts

```diff
@@ -96,7 +98,25 @@ const createHashEntity = async (params: {
     },
   );
 
-  await HashEntity.createMultiple(
+  // TODO: allow creating policies alongside entity creation
+  //   see https://linear.app/hash/issue/H-4622/allow-creating-policies-alongside-entity-creation
+  const entityUuid = extractEntityUuidFromEntityId(entity.entityId);
+  await createPolicy(graphApiClient, params.authentication, {
+    name: `linear-synced-administer-entity-${entityUuid}`,
+    effect: "permit",
+    principal: {
+      type: "actor",
+      actorType: "machine",
+      id: params.authentication.actorId,
+    },
+    actions: ["viewEntity"],
```

Comment:
I believe this should be whatever the `administrator` equivalent is, as that's what we're adding on top of the web defaults above.

Reply:
Same as [below](https://github.com/hashintel/hash/pull/7254#discussion_r2121623877)